client/content-registry/ContainerRegistry.js

```diff
@@ -0,0 +1,42 @@
+import containerList from 'shared/custom-containers';
+import { getContainerName } from 'tce-core/utils';
+import pick from 'lodash/pick';
+import Promise from 'bluebird';
+
+const EXTENSIONS_LIST = 'index';
+
+export default class ContainerRegistry {
```

Comment:
I'm getting awfully nitpicky here, but this class is almost the same as `ElementRegistry` so how about either making `Registry` class and have those two inherit it or extract some of the logic/methods to shared `helpers` file?